docs/designs/output.md

```diff
@@ -26,17 +26,27 @@ For *dynamic rendering*, `docfx` produces an output for lookup by the following
 https://docs.microsoft.com/en-us/dotnet/api/system.string?view=netstandard-2.0#Instantiation
 ```
 
-For *static rendering*, `docfx` produces an output for static file servers using the following URL schema:
+### Static rendering URL schema
 
 `https://{host}/{locale?}/{moniker?}/{site-url}`
 
+#### Static rendering with pretty URL
+
 ```
              host          locale    moniker            site-url
         |------^---------| |-^-| |------^------||----------^-------------|
 https://docs.microsoft.com/en-us/netstandard-2.0/dotnet/api/system.string/#Instantiation
 ```
 
-> This document uses pretty url format that ends with `/`. We also supports ugly url format that ends with `.html`.
+#### Static rendering with ugly URL
```

Comment:
Rename `ugly URL` to `normal URL`? Just don't want to name the default behavior "ugly"...

Reply:
It's what hogo uses.

Reply:
got it.